src/LightningExtension/Context/EntityBrowserContext.behat.inc

```diff
@@ -106,4 +107,40 @@ class EntityBrowserContext extends DrupalSubContextBase {
     $items[$n - 1]->click();
   }
 
+  /**
+   * Asserts a certain number of items are visible in the entity browser.
+   *
+   * @param int $n
+   *   The number of media items that should be visible.
+   * @param string $browser_id
+   *   (optional) The entity browser ID.
+   *
+   * @throws ExpectationException
+   *   If actual number of in the entity browser does not match the expected.
+   *
+   * @Then I should see :n items in the entity browser
+   */
+  public function assertNumberVisible(int $n, $browser_id = NULL) {
+    if ($browser_id) {
+      $selector = 'form#entity-browser-' . Html::cleanCssIdentifier($browser_id) . '-form';
+    }
+    else {
+      $selector = 'form[data-entity-browser-uuid]';
+    }
+
+    /** @var \Behat\Mink\Element\NodeElement[] $items */
+    $items = $this
+      ->assertSession()
+      ->elementExists('css', $selector)
+      ->findAll('css', '[data-selectable]');
+    $actual = count($items);
```

Comment:
All of the preceding logic should be moved into a protected method and re-used, since it's copied directly from selectItem() and we should probably not be repeating ourselves :)